index.html

```diff
@@ -266,8 +281,10 @@ <h3 class="heading5">Professional Art Printing Data</h3>
           <a class="icon2" href="#"><img src="images/ruby1.png" alt="ruby" /></a>
         </li>
       </ul>
-
-      <button class="button2">See Project</button>
+      <div class=buttonlink onclick="modal(3)">
+        <button class="button2">See Project</button>
+      </div>
+      
     </section>
```

Comment:
- [ ] Starting from line `144 up to line 288`, this section contains the work cards HTML code, please remove them from the HTML file and generate them dynamically using javascript as per the requirement bellow

<img width="1031" alt="3" src="https://user-images.githubusercontent.com/77843050/210660442-a080dd88-e79d-48ac-9fcc-225d13d9231f.png">